src/components/Input/Input.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Input.scss';
+
+function Input(props) {
+  const { type, className, name, placeholder } = props;
```

Comment:
- 함수 컴포넌트의 선언 방식도 통일하면 좋을 것 같습니다.
- props는 매개변수 자리에서도 받아올 수 있습니다.
```suggestion
const Input = ({ type, className, name, placeholder }) => {
```